packages/backend/src/peripherals/database/BalanceRepository.ts

```diff
@@ -51,6 +51,7 @@ export class BalanceRepository extends BaseRepository {
       .insert(rows)
       .onConflict(['block_number', 'holder_address', 'asset_id'])
       .merge()
+      .returning('block_number')
```

Comment:
Why is this necessary?

Reply:
Reverted this changed and fixed it properly in #484